tests/system/View/TableTest.php

```diff
@@ -375,6 +434,55 @@ public function testGenerate()
         $this->assertStringContainsString('<td>12345</td>', $table);
     }
 
+    public function testGenerateWithStyle()
+    {
+        $template = [
+            'table_open'         => '<table border="1" cellpadding="4" cellspacing="0">',
+            'thead_open'         => '<thead>',
+            'thead_close'        => '</thead>',
+            'heading_row_start'  => '<tr>',
+            'heading_row_end'    => '</tr>',
+            'heading_cell_start' => '<td>',
+            'heading_cell_end'   => '</td>',
+            'tfoot_open'         => '<tfoot>',
+            'tfoot_close'        => '</tfoot>',
+            'footing_row_start'  => '<tr>',
+            'footing_row_end'    => '</tr>',
+            'footing_cell_start' => '<td>',
+            'footing_cell_end'   => '</td>',
+            'tbody_open'         => '<tbody>',
+            'tbody_close'        => '</tbody>',
+            'row_start'          => '<tr>',
+            'row_end'            => '</tr>',
+            'cell_start'         => '<td>',
+            'cell_end'           => '</td>',
+            'row_alt_start'      => '<tr>',
+            'row_alt_end'        => '</tr>',
+            'cell_alt_start'     => '<td>',
+            'cell_alt_end'       => '</td>',
+            'table_close'        => '</table>',
+        ];
+
+        $this->table->setTemplate($template);
+        $this->table->setHeading(['Name', ['data' => 'Amount', 'style' => $this->styleTable]]);
+
+        $this->table->addRow(['Fred', 1]);
+        $this->table->addRow(['Mary', 3]);
+        $this->table->addRow(['John', 6]);
+
+        $this->table->setFooting(['Total', ['data' => '<small class="text-light">IDR <span class="badge badge-info">10</span></small>', 'style' => $this->styleTable]]);
```

Comment:
The same style is used. But it is better to change the style for footer so that we can confirm that `setFooting()` really changes the style.

Reply:
![image](https://user-images.githubusercontent.com/97607754/189523451-dbf345c7-0b9e-40ec-8ee7-3043498e5000.png)

Okay, updated with different style